generatedocs/data/groups/groups.json

```diff
@@ -55,7 +55,8 @@
             "elasticshareapp",
             "elasticuploadapp",
             "uploaddata",
-            "elasticmoveapp"
+            "elasticmoveapp",
+            "elasticpublishapp"
```

Comment:
The list of actions should be alphabetically ordered. Please move "elasticpublishapp" (and "elasticmoveapp") to the correct place in the list.